package.json

```diff
@@ -108,6 +108,9 @@
         "react-router-dom": "^5.2.0"
     },
     "dependencies": {
+        "@react-aria/checkbox": "^3.2.2",
+        "@react-aria/focus": "^3.4.0",
+        "@react-stately/toggle": "^3.2.2",
```

Comment:
You don't have to install them separately, `react-aria` and `react-stately` are already installed if you rebase